oio/cli/object/object.py

```diff
@@ -187,7 +188,11 @@ def take_action(self, parsed_args):
                         mime_type=parsed_args.mime_type,
                         autocreate=autocreate)
 
-                    results.append((name, data[1], data[2].upper(), 'Ok'))
+                    res = [name, data[1], data[2].upper(), 'Ok']
+                    if perfdata:
+                        res += (perfdata,)
```

Comment:
No need to wrap this into a tuple: the element will be added to the list anyway.
Or we can make both elements tuples:
```
res = (name, data[1], data[2].upper(), 'Ok')
if perfdata:
    res = res + (perfdata, )
```